test/public_test.rb

```diff
@@ -64,7 +72,12 @@ class Songs < Cell::Concept
   # #cell(collection: [], genre: "Fusion").() doesn't change options hash.
   it do
     Cell::ViewModel.cell("public_test/song", options = { genre: "Fusion", collection: [Object] }).()
-    _(options.to_s).must_equal "{:genre=>\"Fusion\", :collection=>[Object]}"
+
+    if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('3.4.0')
+      _(options.to_s).must_equal "{:genre=>\"Fusion\", :collection=>[Object]}"
```

Comment:
Please use https://github.com/trailblazer/trailblazer-core-utils/blob/master/lib/trailblazer/core/utils/inspect.rb#L4 for `Hash#inspect` :beers: